app/controllers/PenaltiesFrontendController.scala

```diff
@@ -43,37 +40,13 @@ class PenaltiesFrontendController @Inject()(etmpService: ETMPService,
                                cc: ControllerComponents)
   extends BackendController(cc) {
 
-  def getPenaltiesData(enrolmentKey: String, arn: Option[String] = None, newApiModel: Boolean = false,
-                       newFinancialApiModel: Boolean = false): Action[AnyContent] = Action.async {
+  def getPenaltiesData(enrolmentKey: String, arn: Option[String] = None): Action[AnyContent] = Action.async {
     implicit request => {
-      if(!newApiModel) {
-        etmpService.getPenaltyDataFromETMPForEnrolment(enrolmentKey).map {
-          result => {
-            result._1.fold {
-              result._2 match {
-                case Left(GetETMPPayloadNoContent) => NotFound(s"Could not retrieve ETMP penalty data for $enrolmentKey")
-                case _ => InternalServerError("Something went wrong.")
-              }
-            }(
-              etmpData => {
-                if (etmpData.pointsTotal > 0) {
-                  val auditModel = UserHasPenaltyAuditModel(etmpData, RegimeHelper.getIdentifierFromEnrolmentKey(enrolmentKey),
-                    RegimeHelper.getIdentifierTypeFromEnrolmentKey(enrolmentKey),
-                    arn)
-                  auditService.audit(auditModel)
-                }
-                Ok(Json.toJson(etmpData))
-              }
-            )
-          }
-        }
-      } else {
-        handleGetPenaltyDetailsCall(enrolmentKey, arn, newFinancialApiModel)
+        handleGetPenaltyDetailsCall(enrolmentKey, arn)
```

Comment:
Do we need the handleGetPenaltyDetailsCall method? Is it best to add the body of that method in here